ixwebsocket/IXWebSocketMessageQueue.cpp

```diff
@@ -33,12 +33,12 @@ namespace ix
         {
             // set dummy callback just to avoid crash
             _websocket->setOnMessageCallback([](
-                WebSocketMessageType type,
-                const std::string & str,
-                size_t wireSize,
-                const WebSocketErrorInfo & errorInfo,
-                const WebSocketOpenInfo & openInfo,
-                const WebSocketCloseInfo & closeInfo)
+                WebSocketMessageType,
```

Comment:
Nitpick style comment / there should be no space between type and &.

Reply:
Do you use some tool to check style?